bucket/raine.json

```diff
@@ -0,0 +1,75 @@
+{
+    "version": "0.93.5d",
+    "description": "RAINE is an emulator for arcade games",
+    "homepage": "https://raine.1emulation.com/",
+    "license": "Artistic-2.0",
+    "notes": [
+        "",
+        "Download the corresponding DLLs from http://raine.1emulation.com/download/latest.html - basically for either 32-bit or 64-bit Windows operating systems.",
+        "",
+        "Once downloaded, extract all of them to the System32 dir under your Windows dir.",
```

Comment:
I'm not fond of this approach, as it requires administrator privileges to work. Can't the libraries be put in the same folder as the executable?

Reply:
It could, but I couldn't find a reliable way to `checkver`. Open to suggestions.

Reply:
Same as ref-vk for yquake2 - updating commit.